docs/command-line-syntax.md

```diff
@@ -36,11 +36,19 @@ _*example:*_
 Ellipsis represent arguments that can appear multiple times
 
 _*example:*_
-`gh pr close <numbers … >`
+`gh pr close <numbers>...`
 
 ## Variable naming
 
 For multi-word variables use dash-case (all lower case with words seperated by dashes)
 
 _*example:*_
 `gh pr checkout <issue-number>`
+
+## Complex examples
+
+_*required argument with mutually exclusive optoins:*_
+`gh pr close <{number | url}>`
```

Comment:
I don't think there is ever a precedent for putting anything other than a placeholder name inside a `<...>` (i.e. no expressions). So I would suggest writing `{<number> | <url>}` here (i.e. braces expression wraps the placeholders, not the other way around). That's how we did it so far and I find it more readable, even if it has more characters overall!

Reply:
You're correct! `{<number> | <url>}` is what should be here.